### ISSUE TITLE
How to map multiple `rightsHolder`

### ISSUE THREAD

**Issue by peterdesmet** [inbo/esas2obis#32]
@pieterprovoost @rubenpp7 some campaigns in ESAS have **multiple dataRightsHolders**, e.g.:

https://esas.ices.dk/api/getCampaignRecords?campaignID=775
```json
[
  {
    "tblUploadID": 231,
    "tblCampaignID": 208521,
    "dataRightsHolder": "3269~2299",
    "country": "DE",
    "campaignID": "775",
    "dataAccess": "Public",
    "startDate": "2011-07-27",
    "endDate": "2011-08-22",
    "notes": null
  }
]
```

`3269~2299` indicates `Federal Agency for Nature Conservation (BfN) | Research and Technology Centre (Buesum) (FTZ)`

Can I map both in `rightsHolder` (separated with `|`) or do you prefer that I only keep the first one?

**Comment by peterdesmet** [comment]
Sorry, submitted before issue was finished. Updated now in issue body.

**Comment by peterdesmet** [comment]
@nicolasvanermen prefers to list all rightsHolders

**Comment by nicolasvanermen** [comment]
Best is to map all data rights holders.

**Comment by pieterprovoost** [comment]
Yes, pipe separated rightsHolders is fine.

**Comment by peterdesmet** [comment]
Fixed in 0a6c8fa
